app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -55,13 +57,18 @@ class HomeFragmentPresenter @Inject constructor(
       EXPLORATION_ID
     ).observe(fragment, Observer<AsyncResult<Any?>> { result ->
       when {
-        result.isPending() -> logger.d("HomeFragment", "Loading exploration")
-        result.isFailure() -> logger.e("HomeFragment", "Failed to load exploration", result.getErrorOrNull()!!)
+        result.isPending() -> logger.d(TAG_HOME_FRAGMENT, "Loading exploration")
+        result.isFailure() -> logger.e(TAG_HOME_FRAGMENT, "Failed to load exploration", result.getErrorOrNull()!!)
         else -> {
-          logger.d("HomeFragment", "Successfully loaded exploration")
+          logger.d(TAG_HOME_FRAGMENT, "Successfully loaded exploration")
           routeToExplorationListener.routeToExploration(EXPLORATION_ID)
         }
       }
     })
   }
+
+  fun topicButtonClick(v: View) {
```

Comment:
Please remove this function, if it is unused.

Reply:
Its used in xml